### PR TITLE
Fixing first run database schema

### PR DESCRIPTION
**Pull request by galonias**
I really loved your solution and will be switching from https://github.com/EvTheFuture/SmartCharging to smartcharge.dev

However when I tried setting up the application in Docker, I realized that the database creation didn't work due to the TSQL sequenece. I changed the order of the TSQL and added collation information to the Vehicle table and that seems to have fixed it.

Thanks for your hard work!

### CODE REVIEW

File: INSTALL [hunk 1]
```diff
@@ -15,5 +15,5 @@ psql postgres -c "CREATE DATABASE smartcharge WITH OWNER scserver ENCODING UTF8
 # set our default schema
 psql postgres -c "ALTER ROLE scserver IN DATABASE smartcharge SET search_path TO scserver;"
 # grant?
-# GRANT ALL PRIVILEGES ON DATABASE scserver TO scserver;"
+# GRANT ALL PRIVILEGES ON DATABASE smartcharge TO scserver;"
```
Comment (fredli74) [inline INSTALL:18]: good catch, search and replace FTL

**Comment by fredli74** [comment]
Thanks for this! Fun to know that at least one other person is trying to host his own setup. ;)

**Comment by fredli74** [comment]
Sorry, I just realized I squash merged out of old habits. You will have to reset your fork.  😳

**Comment by galonias** [comment]
No worries @fredli74! I noticed that you've improved the INSTALL file a bit more. Will run a trial for a while until we are switching from Skekraft to Tibber in late September. I also added information in one of the issues about creating a Docker image "quick and dirty"-style, but it works! ;-)

**Pull request merged**